src/test-utils/dom/internal/autosuggest-input.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { ElementWrapper } from '@cloudscape-design/test-utils-core/dom';
+import inputStyles from '../../../input/styles.selectors.js';
+import dropdownStyles from '../../../internal/components/dropdown/styles.selectors.js';
+import styles from '../../../internal/components/autosuggest-input/styles.selectors.js';
+import { InputWrapper } from '../index.js';
+import DropdownWrapper from './dropdown.js';
+
+export default class AutosuggestInputWrapper extends ElementWrapper {
```

Comment:
I guess you wanted to use `ComponentInput` here.

Primary difference

1. ElementInput gets extended with `findSomething` method for each our component (`findButton`, `findCheckbox`, `findInput` and so on)
2. ComponentWrapper does not